### PR TITLE
OP-144/Sylius upgrade

### PR DESCRIPTION
**Pull request by rafalmal**

### CODE REVIEW

File: tests/Application/package.json.~1.11.0.dist [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
```
Comment (senghe) [inline tests/Application/package.json.~1.11.0.dist:1]: The documentation update is necessary for the file-split.

**Review by senghe — CHANGES_REQUESTED** [review]

File: tests/Application/config/packages/framework.yaml [hunk 2]
```diff
@@ -3,4 +3,9 @@ framework:
     form: true
     csrf_protection: true
     session:
+#        handler_id: 'session.handler.native_file'
```
Comment (senghe) [inline tests/Application/config/packages/framework.yaml:6]:
```suggestion
```

**Review by senghe — CHANGES_REQUESTED** [review]
The file FrankMartinShippingExportEventListener.php has not resolved comments

**Review by senghe — APPROVED** [review]

File: tests/Behat/Mock/EventListener/FrankMartinShippingExportEventListener.php [hunk 3]
```diff
@@ -12,21 +12,21 @@
 
 use BitBag\SyliusShippingExportPlugin\Entity\ShippingExportInterface;
 use Doctrine\Persistence\ObjectManager;
-use Sylius\Bundle\CoreBundle\Provider\FlashBagProvider;
+use Sylius\Bundle\CoreBundle\Provider\requestStackProvider;
```
Comment (andriusvo) [inline tests/Behat/Mock/EventListener/FrankMartinShippingExportEventListener.php:15]: This won't work as it class name starts from lower case

File: tests/Behat/Mock/EventListener/FrankMartinShippingExportEventListener.php [hunk 4]
```diff
@@ -12,21 +12,21 @@
...
 use Sylius\Bundle\ResourceBundle\Controller\FlashHelperInterface;
 use Sylius\Bundle\ResourceBundle\Event\ResourceControllerEvent;
 use Symfony\Component\Filesystem\Filesystem;
 use Symfony\Component\HttpFoundation\RequestStack;
-use Symfony\Component\HttpFoundation\Session\Flash\FlashBagInterface;
+use Symfony\Component\HttpFoundation\Session\Flash\requestStackInterface;
```
Comment (andriusvo) [inline tests/Behat/Mock/EventListener/FrankMartinShippingExportEventListener.php:20]: This won't work as it class name starts from lower case, same here

File: tests/Behat/Mock/EventListener/FrankMartinShippingExportEventListener.php (hunk 3 above)
Comment (rafalmal) [reply tests/Behat/Mock/EventListener/FrankMartinShippingExportEventListener.php]: Thank You for pointing that out, it is fixed now.

**Review by andriusvo — APPROVED** [review]

**Comment by andriusvo** [comment]
@rafalmal @senghe any plans to merge it soon?

**Comment by senghe** [comment]
@andriusvo merged =)

**Pull request merged**